storage-engine/titan-configuration.md

```diff
@@ -131,7 +142,12 @@ To fully disable Titan for all existing and future data, you can follow these st
     discardable-ratio = 1.0
     ```
 
-2. Perform a full compaction using tikv-ctl. This process will consume large amount of I/O and CPU resources.
+    > **Note:**
+    >
+    > When `discardable-ratio=1`, it means TiKV will only recycle a Titan blob file when all its data are moved to RocksDB. That means before the convertion completes, these Titan blob files won't be deleted. And therefore, if a TiKV node does not have sufficent disk size to store both Titan and RocksDB data, the parameter should keep the default value instead of `1.0`. However if the disk size is big enough, `discardable-ratio = 1.0` can help to reduce the blob file GC and the disk IO. 
+    >
```

Comment:
```suggestion
    > Use the default value `0.5` for `discardable-ratio` when there is not enough disk space to hold both Titan and RocksDB data. In general, it is recommended to use the default value if the free disk space is less than 50%. This is because when `discardable-ratio = 1.0`, the RocksDB data keeps growing, and at the same time, the recovery of Titan's original blob file requires all the data in that file to be migrated to RocksDB, which is a slow process. If the disk size is big enough, setting `discardable-ratio = 1.0` can reduce the GC of the blob file itself during compaction, which saves bandwidth.
```